src/engine/strat_engine/blockdev.rs

```diff
@@ -207,7 +207,7 @@ pub fn initialize(pool_uuid: &PoolUuid,
 }
 
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
I will take another look at this and ask @agrover if he has any ideas.

I wasn't able to get StratPool::new() to work without making a clone of the BlockDevs in the Vec.

I think the alternative would be constructing the ThinPoolDev outside of new().  Hopefully I'm missing something.